txmongo/collection.py

```diff
@@ -367,7 +367,7 @@ def __apply_find_filter(spec, c_filter):
 
     @timeout
     def find_with_cursor(self, *args, **kwargs):
-        """find_with_cursor(filter=None, projection=None, skip=0, limit=0, sort=None, **kwargs)
+        """find_with_cursor(filter=None, projection=None, skip=0, limit=0, sort=None, batchSize=0, **kwargs)
```

Comment:
Should be `batchsize`, not `batchSize`. By the way, it should probably be `batch_size` to conform to PEP8.